package.json

```diff
@@ -48,9 +52,10 @@
     "postcss-loader": "^2.0.6",
     "prop-types": "^15.5.10",
     "raw-loader": "^0.5.1",
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0",
+    "react": "16.0.0",
```

Comment:
Do we need to pin these for RN or something?

Reply:
Yeah, `react-native` doesn't accept a range on `react` as a peer dep. Each version of `react-native` seems to be tied to a specific version of `react`.

Reply:
SGTM